Improve instructions

yvanzo opened the pull request:
Those are small improvements and corrections to the current instructions.

See commit messages for details.

yvanzo committed:
Fix instructions to build materialized tables

In the commit d42bced10df99e8206d3f1fc5dca68a229ec2db2, these
instructions have been moved to prevent starting the server without the
materialized tables, but then the services are not up yet.

This patch changes the instructions to make it work even though the
musicbrainz server is not running yet.

yvanzo committed:
Document troubleshooting RabbitMQ’s access refused

yvanzo committed:
Document that search updates are asynchronous

reosarevok reviewed (approved):
Changes seem good.

I understand given the materialized tables are built as a one-off, it doesn't matter which mbs container runs the script anyway, so the first commit just does the same in a way that should always work - which makes me wonder why we didn't always do it that way, but that's not particularly important I guess.

yvanzo committed:
Update the section ‘Customize download server’

There is only one download server available nowadays, hence customizing
download server is currently useless but for development purposes.

Reference: https://metabrainz.org/datasets/download

yvanzo committed:
Revise 21c9b4f6f: mirror.log is not always rotated

The replication log file `mirror.log` can be rotated and available at
`mirror.log.1` but it is not (or no longer?) systematically rotated
after the replication is done. This issue is still under investigation,
but at least the documentation is no longer misleading.

yvanzo commented:
All the issues reported by @nelgin through https://github.com/metabrainz/musicbrainz-docker/issues/301 have been addressed by the above commits as far as I can tell.
